src/FufillerLogo.jsx

```diff
@@ -61,11 +74,12 @@ export default class FulfillerLogo extends React.Component {
           throw response;
         }
       }).then(blob => {
-        this.setState({imageBlob: blob});
+        this.setState({imageObjectUrl: URL.createObjectURL(blob)})
```

Comment:
It's so misleading that this is called 'objectUrl' - I assume it's a simple string (that points to an object) with a couple of decorators. And in reality it's a file! I wish it was named differently. :(

Just to make it clear - I'm not complaining that it's named that way in our code, as we're adhering to the established standard. I'm unhappy that the standard named it in this way. :) BLOB tells me a lot more. :)